pkg/kubectl/service.go

```diff
@@ -57,7 +57,7 @@ func paramNames() []GeneratorParam {
 		// port will be used if a user specifies --port OR the exposed object
 		// has one port
 		{"port", false},
-		// ports will be used iff a user doesn't specify --port AND the
+		// ports will be used if a user doesn't specify --port AND the
```

Comment:
Thank you for review. I see it. I suggest that if this is a iff case, use "if and only if" is better. "iff" is really stranger for non-native speaker. After all writing down "iff" will show error in doc, even in github input box. If it is not a iff case, just use "if". Thanks.